dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBean.java

```diff
@@ -98,17 +107,39 @@ public ExternalTool save(ExternalTool externalTool) {
         return em.merge(externalTool);
     }
 
+    // Used only in REST
```

Comment:
IMO this comment is not helpful. We can check where it is being used in IDE, but it'll be hard to maintain if we use this method elsewhere. If it's meant to be used only in REST maybe change the comment to `Should be only used in REST`, if not just remove it pls.